DataFormats/L1TParticleFlow/src/classes_def.xml

```diff
@@ -0,0 +1,45 @@
+<lcgdict>
+                      
+  <class name="l1t::PFCluster" ClassVersion="5">
+        <version ClassVersion="5" checksum="1668592434"/>
+        <version ClassVersion="4" checksum="387028836"/>
+        <version ClassVersion="3" checksum="2630618461"/>
```

Comment:
The convention for new classes (that have never been stored) is to not include any development-time intermediate versions (i.e. start directly with class version 3).

Reply:
done.